src/Microsoft.AspNet.Mvc.Razor.Host/Directives/ChunkInheritanceUtility.cs

```diff
@@ -25,8 +25,8 @@ public class ChunkInheritanceUtility
         /// <summary>
         /// Initializes a new instance of <see cref="ChunkInheritanceUtility"/>.
         /// </summary>
-        /// <param name="razorHost">The <see cref="MvcRazorHost"/> used to parse _ViewStart pages.</param>
-        /// <param name="codeTreeCache"><see cref="ICodeTreeCache"/> that caches _ViewStart <see cref="CodeTree"/>
+        /// <param name="razorHost">The <see cref="MvcRazorHost"/> used to parse _GlobalImport pages.</param>
```

Comment:
Below in the `ViewHierarchyUtility.cs` you do `<c>_GlobalImport.cshtml</c>` when mentioning `_GlobalImport`, should be consistent one way or the other.